hash/pico.xml

```diff
@@ -4928,7 +4928,7 @@ But how do later protos fit with this theory? Maybe the later protos were from t
 		<publisher>Sega Toys</publisher>
 		<info name="serial" value="HPC-6114"/>
 		<info name="release" value="200207xx"/>
-		<info name="alt_title" value="トミカ・ピコ レスキューパーキング きんきゅうしゅつどう! はたらくのりもの &lt;初回･トミカ付属&gt; - Tomica Pico Rescue Parking Kinkyuushutsudou! Hataraku Norimono ~Shokai-Tomica Fuzoku~"/>
+		<info name="alt_title" value="トミカ・ピコ レスキューパーキング きんきゅうしゅつどう! はたらくのりもの &lt;初回・トミカ付属&gt; - Tomica Pico Rescue Parking Kinkyuushutsudou! Hataraku Norimono ~Shokai-Tomica Fuzoku~"/>
```

Comment:
Why is the transliteration in the description as well as being appended to the Japanese title here?  I don’t think it needs to be in both.

Reply:
No, it isn't on the box (as one would expect): https://i.ebayimg.com/images/g/5HIAAOSw3J1h~C3x/s-l1600.jpg